script/modify_needle

```diff
@@ -31,14 +31,15 @@ print help
 
 =cut
 
+use Feature::Compat::Try;
 use Mojo::Base -strict, -signatures;
 use Mojo::File qw(path);
 use Cpanel::JSON::XS ();
 use Getopt::Long;
 
 sub usage ($r) {
-    eval "use Pod::Usage; pod2usage($r);";
-    die "cannot display help, install perl(Pod::Usage)\n" if $@;    # uncoverable statement
+    try { use Pod::Usage; pod2usage($r) }
+    catch ($e) { die "cannot display help, install perl(Pod::Usage)\n" }
```

Comment:
Actually, we don't need the try/catch or eval here, see a0a409dbbcc73646f3d9d9551bd71de940b7bf0f
I don't know why I missed that single script

Reply:
done